README.md

```diff
@@ -1,100 +1,80 @@
-﻿# liberation-circuit
+# Liberation Circuit
 
-version: 1.3
-
-This the release version of Liberation Circuit, an RTS/programming game.
+This is the release version of Liberation Circuit, an RTS/programming game.
 
 To play the prebuilt binaries on Windows, [download the latest release](https://github.com/linleyh/liberation-circuit/releases) and run `LibCirc.exe`.
 
+## Screenshots
 
-It should compile on any OS supported by Allegro 5 - to build, compile the c files in the source directory and link with Allegro 5. More detailed instructions are below. More detail about the source file structure is at the start of m_main.c.
-
-The executable should go in the "bin" subdirectory (the same directory as the "init.txt" file). The game requires write access to this directory to save mission progress. If this isn't okay, you can specify a path in the fopen calls at about lines 2808 and 2860 of h_story.c.
-
-Don't try to compile the .c files in the /proc or /story subdirectories! They are code used by the game itself.
-
-- Manual.html has extensive detail about the game, including documentation for the in-game API.
-
-- Edit init.txt to set screen resolution and other options (fullscreen, sound volume, key rebinding, colourblind mode etc).
-
-It looks like this (this is github markdown):
-
-![a screenshot](http://i.imgur.com/pPIJ03I.png)
-
-![another screenshot](http://i.imgur.com/QKWzkqA.png)
-
+![a screenshot](https://i.imgur.com/pPIJ03I.png)
```

Comment:
Quick question - is it sensible to host images for the README file on imgur? How stable/reliable is that URL?

Reply:
Assuming the account isn't deleted, the image should remain persistent until imgur shuts down. If reliability is an issue, screenshots could be stored within the repository.